enumbler.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activerecord', ['>= 5.2.3', '< 6.1']
   spec.add_dependency 'activesupport', ['>= 5.2.3', '< 6.1']
 
+  spec.add_development_dependency 'byebug'
```

Comment:
If I had my pick I would go with `pry` over byebug.  Seems much more powerful and you can stop execution in the same way using `binding.pry`.  They even have a byebug-like setting so you can use similiar commands.

Either way need to specify an actual version of whatever we go with ...